src/language/class-modifiers.md

```diff
@@ -223,27 +235,29 @@ return switch (Vehicle vehicle) {       // ERROR: The switch is missing a subtyp
 };
 ```
 
-If [exhaustiveness checking][exhaustive] is not necessary for your use case, use
-[`final`](#final) instead.
+If you don't intend users to be able to [exhaustively switch][exhaustive] over
+the subtypes of the class, or you want the freedom to add new subtypes later
+without it being a breaking API change, use [`final`](#final) instead.
```

Comment:
**issue:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).
```suggestion
If you don’t want [exhaustive switching]][exhaustive] or want to add subtypes later,
use [`final`](#final).
```